WS2012R2/lisa/xml/CoreTests.xml

```diff
@@ -292,7 +292,7 @@
 			<files>remote-scripts/ica/modules_check.sh</files>
 			<testparams>
 				<param>TC_COVERED=CORE-24</param>
-				<param>hv_modules=(hv_vmbus.ko hv_storvsc.ko hv_netvsc.ko hyperv-keyboard.ko)</param>
+				<param>hv_modules=(hid-hyperv.ko hv_vmbus.ko hv_storvsc.ko hv_netvsc.ko hyperv_fb.ko hyperv-keyboard.ko)</param>
```

Comment:
Thank you for the addition on dracut-network!

For this test case we are looking at the minimal components that would ensure a Linux VM continues to work.
hid-hyper-v - mouse input and hyperv_fb are not mandatory components, so for that reason we have not included them.